packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md

```diff
@@ -8064,6 +8064,7 @@ export interface ITextFieldProps extends React.AllHTMLAttributes<HTMLInputElemen
     prefix?: string;
     readOnly?: boolean;
     resizable?: boolean;
+    showRevealPassword?: boolean;
```

Comment:
@ecraig12345 naming-wise, are you ok with this name? Is it consistent with any precedents? Do you have other suggestions?

Reply:
Some other ideas:

allowPasswordVisibilityToggle
canRevealPassword

Reply:
IE/Edge do call it Reveal Password internally. canRevealPassword is better than what I have.